code/style.css

```diff
@@ -1,22 +1,60 @@
 * {
   box-sizing: border-box;
+  text-transform: uppercase;
 }
 
 body {
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
+padding-left: 0.3em;
+  font-family: "skolar-sans-latin-condensed", sans-serif;
+  font-weight: 600;
+  text-transform: uppercases;
+  font-style: normal;
+    background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+    background-size: 400% 400%;
+    animation: gradient 15s ease infinite;
+    height: 8S0vh;
 }
+  @keyframes gradient {
+    0% {
+      background-position: 0% 50%;
+    }
+    50% {
+      background-position: 100% 50%;
+    }
+    100% {
+      background-position: 0% 50%;
+    }
+  }
+  
+
 
 h1 {
-  font-weight: bold;
-  font-size: 44px;
-  line-height: 34px;
+  
+  font-family: "fakir-new", sans-serif;
+  font-weight: 700;
+  font-style: italic;
+  font-size: 6em;
+  color: #fff;
+  text-align: center;
+  margin-block-start: 0.1em;
+  margin-block-end: 0.1em;
+  margin-bottom: -0.2em;
+}
+
+.slogan {
```

Comment:
Very good font choice for the slogan as well. You can tell that a lot of thought has been put into the design.